.github/actions/generate_manifest/schema.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 TIER IV, INC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Schema definition for manifest.json."""
+
+from datetime import datetime
+from typing import List, Literal
+
+from pydantic import BaseModel
+
+
+class ReleasePackage(BaseModel):
+    filename: str
+    version: str
```

Comment:
Suggest to define how to represent the patch update path with using the version file.

I suggest we can use the following schema for version field:
normal app img: `x.y.z`, which follows the Semantic Versioning spec.
patch file: `x.y.z>a.b.c` , `>` char is not used in semantic versioning, so we are safe to use it as separator.

Reply:
Or let's add a new field `metadata`, and place the `base_version` information into that field.

Reply:
Added the local test results in the description.